lib/connectors/createsend.rb

```diff
@@ -0,0 +1,38 @@
+module Connectors
+  class Createsend
```

Comment:
This must not be named Createsend. You will use the gem Createsend sometimes through the process. Isn't this connector named CampainMonitor or something?